docs/ref-manual/cli/options.md

```diff
@@ -130,6 +132,15 @@ An example of an “empty” rule:
 
 _We expect all rules to fail this check._ The exception is the fallback function, which might pass.
 
+2. Assert-Vacuity- checks for each assert command in the rule, whether the assert is vacuously true, namely, even if all the previous preconditions are removed, the assert is always evaluated to true.
```

Comment:
What do you mean by previous preconditions?  Do you mean `require`s?

Reply:
Maybe the best way to say this would be "An assertion is vacuously true if it evaluates to true on every example that reaches it, including those that would normally be ruled out by `require` statements."?

Reply:
> Maybe the best way to say this would be "An assertion is vacuously true if it evaluates to true on every example that reaches it, including those that would normally be ruled out by `require` statements."?

Yes, but also should refer to the if statements the assert is nested in.